google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/ResultSetsTest.java

```diff
@@ -34,11 +41,51 @@
 
   @Test
   public void resultSetIteration() {
```

Comment:
its easier to read test case when they are as explicit as possible. ie - variable names could be more informative (even if long)

also testing edge cases or large values provide surface errors than smaller value.. e.g. - testing a large random bytes value

Reply:
I don't disagree if we're testing the implementation, but the purpose of this change is to expand the line coverage, i.e to ensure execution of paths. For edge case testing etc., we would need to add a lot more specific cases for each data type.

Reply:
i was hoping to shoot 2 birds with 1 stone, though i don't mean or need to have extensive edge case testing with this exercise either.